### PR TITLE
Docs: User/Code/Dev

### PR DESCRIPTION
**Pull request by giovaniceotto**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`python setup.py install`) was run locally and any changes were pushed
- [x] Lint (`black rocketpy`) has passed locally and any fixes were made for failures
- [x] All tests (`pytest --runslow`) have passed locally

## Pull request type
 
 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- Documentation does not show `__init__` constructor docstrings as reported in #65.
- Documentation is not organized in an adequate manner for the pydata theme being used.

## What is the new behavior?

- Documentation for all classes now show `__init__` constructor docstrings fixing issue #65. It is worth noting that the `__init__` docstring appears after the class docstring, which may be confusing.
- New organization divides the documentation into three folders:
  - User documentation (example notebooks, such as Getting Started)
  - Code documentation (documentation for all the classes)
  - Development documentation (under construction) 
- ReadMe now contains links to each of the documentation folders
- ReadMe _Main Features_ section was altered to make it occupy less vertical space using drop downs. 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

- Documentation preview available at: [https://rocketpyalpha.readthedocs.io/en/docs-organize_toctree/](https://rocketpyalpha.readthedocs.io/en/docs-organize_toctree/)

### CODE REVIEW

**Comment by giovaniceotto** [comment]
@PatrickSampaioUSP, please wait until you review this PR. I just noticed that I may have introduced a change that causes the Colab Getting Started notebook to fail.

**Comment by giovaniceotto** [comment]
@PatrickSampaioUSP All good, ready to be reviewed and merged.

File: README.md [hunk 1]
```diff
@@ -7,33 +7,77 @@
...
+<ul>
+  <li>Test the exact code that will fly</li>
+  <li>Sensor data can be augmented with noise</li>
+</ul>
+</details>
+
+<details>
+<summary>Solid motors models</summary>
+<ul>
+  <li>Burn rate and mass variation properties from thrust curve</li>
```
Comment (PatrickSampaioUSP) [inline README.md:51]: Perhaps would be interesting to add that currently we only support a SolidMotor with a grain BATES.

File: README.md [hunk 2]
```diff
@@ -7,33 +7,77 @@
...
-  - Weather forecasts and reanalysis
-  - Weather ensembles
-- Monte Carlo simulations for dispersion and sensitivity analysis
-- Flexible and modular
-  - Straightforward engineering analysis (e.g. apogee and lifting off speed as a function of mass)
-  - Non-standard flights (e.g. parachute drop test from helicopter)
-  - Multi-stage rockets
-  - Custom continuous and discrete control laws
-  - Create new classes (e.g. other types of motors)
+### Main features
```
Comment (PatrickSampaioUSP) [inline README.md:10]: I think now it's way better 👌

File: docs/conf.py [hunk 3]
```diff
@@ -66,21 +67,14 @@
...
-            "icon": "fab fa-github-square",
-        },
-        {
-            "name": "LinkedIn",
-            "url": "https://www.linkedin.com/company/rocketpy/",
-            "icon": "fab fa-linkedin",
-        },
-    ],
+  "logo_link": "index",
+  "github_url": "https://github.com/Projeto-Jupiter/RocketPy",
```
Comment (PatrickSampaioUSP) [inline docs/conf.py:75]: A nice idea for a next iteration would be to add the LinkedIn link

**Review by PatrickSampaioUSP — APPROVED** [review]

**Pull request merged**